geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/eventing/BlockingCommandListenerTest.java

```diff
@@ -58,7 +58,7 @@ public void testTimeoutIsAdjusted() {
     verify(context, times(1)).resubmitCommand(argumentCaptor.capture());
 
     double timeout = Coder.bytesToDouble(argumentCaptor.getValue().getCommandArguments().get(0));
-    assertThat(timeout).isLessThan(1.0D);
+    await().untilAsserted(() -> assertThat(timeout).isLessThan(1.0D));
```

Comment:
Since `timeout` is only calculated once, this await will not do what you want. I think you're better off just putting a sleep before the `resubmitCommand()` call.

Reply:
Ah. Interesting point. Yeah, we could try this. At this point as we cannot reproduce we can start with this approach and see.